bytecomp/lambda.ml

```diff
@@ -264,9 +264,25 @@ and lambda_event_kind =
   | Lev_function
   | Lev_pseudo
 
+type module_map = {
+  map :  (int * Ident.t) list;
+  prims : (int * string) list;
+  methcache : int option;
```

Comment:
The cache of methods is stored in the toplevel structure (at the last position), thus, it is a root that needs to be identified.